core/domain/stats_jobs_one_off.py

```diff
@@ -214,6 +214,26 @@ def entity_classes_to_map_over(cls):
 
     @classmethod
     def prepare_map(cls, item):
+        """Returns a dict that represents the given model instance, so that it
+        can be processed by the MapReduce pipeline.
+
+        Args:
+            item: [StateCompleteEventLogEntryModel|
```

Comment:
Drop outer brackets ("[", "]") -- could give the mistaken impression it's a list.

Reply:
Done